### DISCUSSION TITLE
Reverse DNS with spamrats

### DISCUSSION THREAD

**Discussion by cociweb** [docker-mailserver/docker-mailserver#3675] [Q&A] [answered]
Hello,
I'm using this super-perfect mailserver stack for years. Congrats and thank's.

My question is a config related question:
I'm using it over dynaimc IP (without correct reverseDNS) with a very inflexible ISP. My mailing blacklist check pass on every provider except on [spamrats' reverse DNS check](https://www.spamrats.com/rats-dyna.php). They take the reverse-dns misconfiguration as a very serious violence, and there is no possibility to get on whitelist permanently. - This increases my spampoints a little, but not very seriously.
Is there any hint/workaround/way for hack -only- for this service-provider/robot-spider (without screw-up my working config)? Or it is better to leave this alert as it is?

Thank you for your hints!

**Comment by casperklein** [comment, marked as answer]
>  My mailing blacklist check pass on every provider 

Then you are really lucky. Most blacklists have listed the IP ranges from providers given to users (for good reason IMO).

There is no hack or something I am aware of.

Also keep in mind, that many mailserver reject mails coming from a server not having a correct PTR record. DMS also provides that [feature](https://docker-mailserver.github.io/docker-mailserver/latest/config/environment/#postfix_reject_unknown_client_hostname), however it's no enabled by default.

I really love that feature, it rejects almost 99% of spam mails without a single false positive since I enabled back in May.

**Reply by cociweb** [reply]
Okay, that's what I afraid. :/
Unfortunatelly, I'm in a deadlock situation with my ISP provider about the PTR.
My example is a +1 example/reason why should it be kept off by default on the receiver side: or at least track this reason. :)

Anyways, thank you for your answer!
